submissions/modupeAdeOnojobi/database/transportationQuery.sql

```diff
@@ -0,0 +1,34 @@
+1. SELECT survived, COUNT(*) FROM accidents WHERE survived = 1 GROUP BY survived;
+// 290
+
+
+2. SELECT survived, COUNT(*) FROM accidents WHERE survived = 0 GROUP BY survived;
+// 424
```

Comment:
The total number of record is `891` it means when you add `290 + 424`, you ought to have `891` in total. Kindly rerun your query. drop the database, run the query that creates the database and tables again. Then import the passengers CSV file first, then travel detail, before the survival file.